src/app/(private)/admin-registration-applications/page.tsx

```diff
@@ -0,0 +1,58 @@
+'use client'
+import PrivateRoute from '@/app/hocs/isAuth'
+import SelectStatus from '@/components/adminRegistrationApplications/selectStatus'
+import { useEffect, useState } from 'react'
+import Pagination from '@/components/adminRegistrationApplications/pagination'
+import { postAdminRegistrationApplications } from '@/modules/adminRegistrationApplications/service'
+import { useTranslation } from 'react-i18next'
+import DataTable from '@/components/adminRegistrationApplications/dataTable'
+import { SortingState } from '@tanstack/react-table'
+interface AdminRegistrationState {
```

Comment:
interface in altında ve üstünde bir satır boşluk bırakırsak düzen açısından daha iyi olur